<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 7.3.2 (Python 3.13.7 on linux)" generated="2025-09-02T16:14:24.147245" rpa="false" schemaversion="5">
<suite id="s1" name="Robot Interactive Console" source="/home/generic/.vscode-oss/extensions/robocorp.robotframework-lsp-1.13.0-universal/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot">
<test id="s1-t1" name="Default Task/Test" line="5">
<kw name="Interpreter Main Loop" owner="MainLoop">
<status status="PASS" start="2025-09-02T16:14:24.156058" elapsed="13.134783"/>
</kw>
<status status="PASS" start="2025-09-02T16:14:24.155820" elapsed="13.135153"/>
</test>
<status status="PASS" start="2025-09-02T16:14:24.147653" elapsed="13.143613"/>
</suite>
<statistics>
<total>
<stat pass="1" fail="0" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat name="Robot Interactive Console" id="s1" pass="1" fail="0" skip="0">Robot Interactive Console</stat>
</suite>
</statistics>
<errors>
<msg time="2025-09-02T16:14:24.145494" level="WARN">Error in file '/home/generic/.vscode-oss/extensions/robocorp.robotframework-lsp-1.13.0-universal/src/robotframework_ls/vendored/robotframework_interactive/robot_interactive_console.robot' on line 4: Singular section headers like '*** Test Case ***' are deprecated. Use plural format like '*** Test Cases ***' instead.</msg>
</errors>
</robot>
